intel_extension_for_transformers/llm/runtime/graph/docs/infinite_inference.md

```diff
@@ -0,0 +1,38 @@
+Infinite Inference
```

Comment:
Shall we highlight the advantage/enhancements to StreamingLLM paper at the beginning so that people know the differences?

Reply:
Added in ff16cbc0042424a149bc0708b5445fbaf50c10af